### DISCUSSION TITLE
Release Policy of MinIO

### DISCUSSION THREAD

**Discussion by tisonkun** [minio/minio#13737] [Q&A] [answered]
I'd like to know how and when MinIO releases a new version. Is it open for access? I failed to find a document for it.

**Comment by harshavardhana** [comment, marked as answer]
@tisonkun there is no policy - we release when we think it's ready to be released.

**Reply by tisonkun** [reply]
Thanks for your reply. It seems there is no version policy also. Then what does "5 Years Long Term Support" means? Does the company provide hotfixes apart from the development/release cycle?

**Comment by harshavardhana** [comment]
> Thanks for your reply. It seems there is no version policy also. Then what does "5 Years Long Term Support" means? Does the company provide hotfixes apart from the development/release cycle?

That is correct @tisonkun
